presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java

```diff
@@ -160,6 +159,24 @@ public Plan plan(Analysis analysis, Stage stage)
         return new Plan(root, symbolAllocator.getTypes());
     }
 
+    private class PlanOptimizerProcessorImplementation
```

Comment:
* static
* s/PlanOptimizerProcessorImplementation/DefaultPlanOptimizerProcessor
 * do we need to name it? it looks like a lambda